debian/changelog

```diff
@@ -1,3 +1,17 @@
+opx-sai-api (36.0.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
```

Comment:
Should we clarify that this is Static L2 VxLAN support, or is all VxLAN support being added here to SAI?

Reply:
I took this from the feature/VxLan branch, will have to ask Atanu. I assumed all his commits into that branch were vetted.

Reply:
This should have all VxLAN support.

Reply:
This should have all VxLAN support.